ArduCopter/mode_flowhold.cpp

```diff
@@ -277,7 +277,7 @@ void ModeFlowHold::run()
 
         // initiate take-off
         if (!takeoff.running()) {
-            takeoff.start_cm(constrain_float(g.pilot_takeoff_alt,0.0f,1000.0f));
+            takeoff.start_m(constrain_float(g.pilot_takeoff_alt * 0.01, 0.0, 10.0));
```

Comment:
not a blocker of course but g.pilot_takeoff_alt should probably be renamed to g.pilot_takeoff_alt_cm at some point

Reply:
Yes, I agree. I was thinking about that and in hindsight I probably should have done that in the renaming. I was just thinking that I would focus on the parameters after.

Reply:
I have added a commit at the end of the PR to do this.